README.md

```diff
@@ -41,12 +40,21 @@ Check out our [FAQ](https://github.com/nextcloud/calendar/wiki/FAQs). If you don
 
 We don't support Internet Explorer 10 and below. Patches for IE9+ are accepted though.
 
-## Nightly builds
+## Maintainers
+
+- [Georg Ehrke](https://github.com/georgehrke)
+- [Thomas Citharel](https://github.com/tcitworld)
+- [and many more](https://github.com/nextcloud/calendar/graphs/contributors)
+
+If you’d like to join, just go through the [issue list](https://github.com/nextcloud/calendar/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+) and fix some. :)   
+We’re also in [#nextcloud-calendar on freenode IRC](https://webchat.freenode.net/?channels=nextcloud-calendar).
 
-Testing is a great way to contribute without having to write source code.
-Although it's straight forward, setting up the development environment requires some knowledge and extra tools on your device.
+We’d like to thank [BrowserStack](https://www.browserstack.com) for providing us with a free subscription.
+
+## Nightly builds / Pre-releases
 
-We provide [nightly builds](https://nightly.portknox.net/calendar/?C=M;O=D) to enable everyone help testing without setting up the development environment.  
+Instead of setting everything up manually, you can just [download the nightly builds](https://nightly.portknox.net/calendar/?C=M;O=D) or [download a pre-release](https://github.com/nextcloud/calendar/releases) instead.
```

Comment:
Nightly build link seems to be dead.

Reply:
Yes, i know. Let me send an email to Steffen